### ISSUE TITLE
Plugin system for ruleset

### ISSUE THREAD

**Issue by Morrydwenn** [Rolisteam/rolisteam#9]
Hello,

Having the possibility to incorporate rulesets would be very usefull.
With a plugin system, the community will develop rulesets and share them.

Combine to the character sheet implementation (planned in the 1.8 roadmap, if I understood correctly), Rolisteam would become much more attractive.

**Comment by obiwankennedy** [comment]
Yeah, the goal is to provide feature to play complete game and sharing scenario. Such as character sheet + picture for any evidence + text note for chapter + mind map for interaction between players...

The new dice system with generic syntax allows to provide the small first step to rulesets thank of aliases.
The plugin system could be really difficult to manage and distribute.

**Comment by Morrydwenn** [comment]
Hello,

Plugin system was maybe not the right term (I'm not a developper), I was meaning a format not hard-coded in rolisteam  but a readable format system (xml ?).
The community could generate those files and make them available on the forum or website of rolisteam.

**Comment by obiwankennedy** [comment]
I may introduce the implementation of #27 request. Like that the ruleset will bring together many informations about the game.

**Comment by obiwankennedy** [comment]
As I have a better view now, the next release will have character sheet, and  custom character states (and of course, custom dice aliases).
At the end, all of these informations will be include into scenario. But I may provide a way to save game data aside.
